src/main/java/de/clearit/kindergarten/appliance/vendor/VendorEditorView.java

```diff
@@ -43,20 +49,56 @@
   public VendorEditorView(VendorEditorModel model) {
     this.model = model;
   }
-
+  
   // Initialisation *********************************************************
   private void initComponents() {
     firstNameField = BasicComponentFactory.createTextField(model.getBufferedModel(VendorBean.PROPERTY_FIRST_NAME));
     lastNameField = BasicComponentFactory.createTextField(model.getBufferedModel(VendorBean.PROPERTY_LAST_NAME));
     phoneNumberField = BasicComponentFactory.createTextField(model.getBufferedModel(VendorBean.PROPERTY_PHONE_NUMBER));
-    vendorNumberField = BasicComponentFactory.createTextField(model.getVendorNumberFieldModel());
+    
+    
     vendorNumberTable = new StripedTable(new VendorNumberTableModel(model.getSelectionInList()));
     vendorNumberTable.setSelectionModel(new SingleListSelectionAdapter(model.getSelectionInList()
         .getSelectionIndexHolder()));
     addVendorNumberButton = new JButton(model.getAction(VendorEditorModel.ACTION_ADD_VENDOR_NUMBER));
     removeVendorNumberButton = new JButton(model.getAction(VendorEditorModel.ACTION_REMOVE_VENDOR_NUMBER));
-  }
+    
+    
+    vendorNumberField = BasicComponentFactory.createTextField(model.getVendorNumberFieldModel());
+    vendorNumberField.addFocusListener(new FocusListener() {

```

Comment:
Das vendorNumberField wurde in der vorherigen Version ja schon in Zeile 52 initialisiert, da kannst Du direkt danach den Focus Listener hinzufügen und brauchst das nicht ganz unten machen.